draft-ietf-mls-architecture.md

```diff
@@ -425,125 +428,105 @@ cryptographic information:
+in any particular order. The one exception is that, because an MLS group has a
 linear history, the members of the group must agree on the order in which
-changes are applied.  Concretely, the group must agree on which MLS Commit
-messages to apply.  There are a variety of ways to achieve this agreement, but
-most of them rely on some help from the Delivery Service.  For example, if a
-Delivery Service provides delivery in the same order to all group members, then
-the members can simply apply Commits in the order in which they appear.
-
-Each Commit is premised on a given state or "epoch" of the group.  The Delivery
-Service must transmit to the group exactly one Commit message per epoch.
-
-Much like the Authentication Service, the Delivery Service can be split between
-server and client components. Achieving the required uniqueness property will
-typically require a combination of client and server behaviors.  For example,
-all of the following examples provide a unique Commit per epoch:
-
-* A "filtering server" Delivery Service where a server rejects all but the
-  first Commit for an epoch and clients apply each Commit they receive.
-
-* An "ordering server" Delivery Service where a server forwards all messages
-  but assures that all clients see Commits in the same order, and clients.
-
-* A "passive server" Delivery Service where a server forwards all messages
-  without ordering or reliability guarantees, and clients execute some secondary
-  consensus protocol to choose among the Commits received in a window.
-
-The MLS protocol provides three important pieces of information
-within an MLSCiphertext message in order to provide ordering:
-
-* The Group Identifier (group ID) to allow for distinguishing the group for
-  which the message has been sent;
-
-* The Epoch number, which represents the number of changes (version) of
-  the group associated with a specific group ID, and allows for
-  lexicographical ordering of messages from different epochs within the
-  same group;
-
-* The Content Type of the message, which allows the Delivery Service to
-  determine the ordering requirement on the message, in particular
-  distinguishing Commit messages from other messages.
-
-The MLS protocol itself can verify these properties. For instance, if the
-Delivery Service reorders messages from a client or provides different clients
-with inconsistent orderings, then clients can put messages back in their proper
-order.  The asynchronous nature of MLS means that within an epoch, messages are
-only ordered per-sender, not globally.
-
-Note that some forms of Delivery Service misbehavior are still possible and
-difficult to detect. For instance, a Delivery Service can simply refuse
-to relay messages to and from a given client. Without some
-sort of side information, other clients cannot generally
-distinguish this form of Denial of Service (DoS) attack.
-
-## Membership knowledge
```

Comment:
Removed this subsection because using MLS in an anonymous way is a very difficult topic and I don't think it's treated fairly here. Using MLS anonymously is more worthy of it's own top-level section with strong guidance.